### ISSUE TITLE
1.16.3 - CE - Final

### ISSUE THREAD

**Issue by MeinKraft** [waidesoper/thevaultCE#116]
- [ ] Boss rooms - remove bedrock, replace Obelisks with Block Placer
- [ ] Re-work spawning. Mobs should be persistent and VaultScaled upon creation NOT in LivingUpdate Events

**Comment by waidesoper** [comment]
Just commented out the code for capes in PlayerEvents

**Comment by MeinKraft** [comment]
Merged GiveBits command into Internal->Bits command, less confusion and no point in two commands
Renamed ParcelBomb to Gift, makes more sense? parcel bomb, terrorism, too soon?
Added BoosterPack command
Added remaining stats, BOOSTERS, BITS, GIFTS, TRADERS

**Comment by waidesoper** [comment]
I don't think we need to remove the opposites, I think if all the positives have an even value, and all the negatives have an odd value that we should get a wider variation of possible modifiers

**Comment by MeinKraft** [comment]
NTS: Vault_General, Vault_Items, Vault_Modifiers,

**Comment by MeinKraft** [comment]
Needs testing

Coop:
does it pull all players into the vault
do mobs spawn around coop players
does eating vault fruit increase timer for all coop players
